taskboot/push.py

```diff
@@ -78,40 +78,66 @@ def heroku_release(target, args):
     # Load queue service
     queue = taskcluster.Queue(config.get_taskcluster_options())
 
-    # Get the list of matching artifacts as we should get only one
-    matching_artifacts = load_artifacts(args.task_id, queue, args.artifact_filter, args.exclude_filter)
-
-    # Push the Docker image
-    if len(matching_artifacts) == 0:
-        raise ValueError(f"No artifact found for {args.artifact_filter}")
-    elif len(matching_artifacts) > 1:
-        raise ValueError(f"More than one artifact found for {args.artifact_filter}: {matching_artifacts!r}")
-    else:
+    updates_payload = []
+
+    parsed_args = []
+
+    bad_parameter_error_message = f"%r doesn't match format 'dyno-name:/path/to/artifact'"
+
+    # Check format of artifacts params
+    for artifact in args.artifacts:
+        colon_number = artifact.count(":")
+
+        if colon_number != 1:
+            raise Exception(bad_parameter_error_message % artifact)
```

Comment:
Please use `.format` here, we avoid using `%` in our Python projects

Reply:
Sure, old habits die hard